pyminiaudio/examples/capture.py

```diff
@@ -0,0 +1,32 @@
+import miniaudio
+from time import sleep
+
+
+if __name__ == "__main__":
+    buffer = b""
+
+    def record_to_buffer():
+        data = yield
+        while True:
+            data = yield
+            print(".", end="", flush=True)
+            global buffer
```

Comment:
I don't think using `global` here is best (it's a bit hacky), but it works for a simple demo.

Reply:
I agree, for a demo program it's okay